### DISCUSSION TITLE
still 404, using nginx with linux panel

### DISCUSSION THREAD

**Discussion by xuakson** [YOURLS/YOURLS#3280] [General]
i need help, my redirect aint working. I'm using aapanel with nginx plugin installed.
I'm able to open the page , mydomain/admin but after creating the shortlink they dont work . says 404.
I havent made any changes to the .htaccess file
https://ibb.co/9twBg8M

**Comment by ozh** [comment]
But since nginx doesn't work with .htaccess ....

Tip : read YOURLS readme about nginx.

**Reply by xuakson** [reply]
ok bosss but please stay here , so i fix with u

**Reply by xuakson** [reply]
is it this sir? https://github.com/YOURLS/YOURLS/wiki/Nginx-configuration

**Reply by dgw** [reply]
That is a great starting point.

**Reply by xuakson** [reply]
https://ibb.co/CPNx0yC
I found the nginx config file.( default)
but i dont know, should i create another ? i'm thinking to create another yourl.conf and placed it at the same location of the default one

**Reply by xuakson** [reply]
my files (yourl) location: https://ibb.co/9twBg8M

**Reply by ozh** [reply]
You're trying to learn nginx, which is certainly a neat idea, but you are in the wrong place to get support to do so. As stated in the YOURLS docs regarding nginx: " if you elect to run nginx you're supposed to be able to adapt the configuration"

Every comment here is sent to 300+ people : please keep discussion about YOURLS itself. Thanks in advance.

**Reply by dgw** [reply]
Learning nginx and/or aapanel at the same time, a tall order.

Web searches should help you find out about where to put the nginx configuration if you want to have separate files for each site, and/or you can ask in [the aapanel forums](https://forum.aapanel.com/) about how to do it via their interface.

**Reply by xuakson** [reply]
ok, not actually wanted to learn , i was tryna host the yourl. so i use that to learn it. but i have little idea on amending the file itself as i did install manually coded url shortner with node js. And it worked but dont know why this one isnt working (yourl). i'm able to acess the admin page and create the domain anyway. You check it , its working : https://ceasa.68ecom.com/admin/, Thanks sir

**Reply by xuakson** [reply]
@dgw ok boss, let me try there too, Thank u

**Reply by xuakson** [reply]
It worked now , i use apache and it did worked fine , thanks bosses, i'm looking up to multi domain feature, i tried and it didnt work
